cmd/argoexec/commands/wait.go

```diff
@@ -61,11 +57,7 @@ func waitContainer(ctx context.Context) error {
 	if err != nil {
 		wfExecutor.AddError(err)
 	}
-	// Annotating pod with output
```

Comment:
This does not seem correct to me. We should not have one annotation per container. Instead, we should have a single output for all containers.